ioBroker/NsPanelTs.ts

```diff
@@ -327,6 +921,18 @@ function SendTime(): void {
     SendToPanel(<Payload>{ payload: "time~" + hr + ":" + min });
 }
 
+function ScreensaverDimmode() {
+    if (timeDimMode.dimmodeOn != undefined ? timeDimMode.dimmodeOn : false) {
+        if (compareTime(timeDimMode.timeNight != undefined ? timeDimMode.timeNight : "23:00", timeDimMode.timeDay != undefined ? timeDimMode.timeDay : "06:00", "not between", null)) {
+            SendToPanel({ payload: "dimmode~" + timeDimMode.brightnessDay});
+        } else {
+            SendToPanel({ payload: "dimmode~" + timeDimMode.brightnessNight}); 
+        }         
+    } else {
+        SendToPanel({ payload: "dimmode~" + config.dimmode });
```

Comment:
ohne genauer geschaut zu haben, denke hier fehlt der zweite wert für die active brightness, in L897 ist er da

Reply:
Stimmt, irgendwie übersehen.